assets/src/styles/blocks/TakeActionBoxout/style.scss

```diff
@@ -152,6 +110,74 @@
   }
 
   @include large-and-up {
-    min-height: 210px;
+    height: 210px;
+    padding: 0;
+
+    img {
+      min-width: 336px;
+      border-top-left-radius: 4px;
+      border-bottom-left-radius: 4px;
+
+      html[dir="rtl"] & {
+        border-top-left-radius: 0;
+        border-bottom-left-radius: 0;
+        border-top-right-radius: 4px;
+        border-bottom-right-radius: 4px;
+      }
+    }
+
+    .boxout-content {
+      padding: 24px;
+    }
+
+    .boxout-excerpt {
+      -webkit-line-clamp: 3;
+    }
+
+    .btn {
+      margin-inline-end: 16px;
+      margin-bottom: 16px;
+    }
+  }
+}
+
+@media screen and (max-width: $large-width) {
+  .sticky-bottom-mobile {
+    z-index: 9001;
```

Comment:
I wonder why `9001`?

Reply:
I actually don't know either 😬 @Inwerpsel wrote that line

Reply:
Just a random high value, you can change it :sweat_smile:

Reply:
Gotcha! 
For me is the same too, but as a convention I would prefer change to something like `z-index: 9999`. it will seem more familiar for developers and we would avoid doubts (like mine 😆).